doc/source/parameters/dem/floating_mesh.rst

```diff
@@ -10,45 +10,37 @@ Floating meshes (solid objects) are finite (limited) auxiliary objects that can
 .. note:: 
     At the moment, solid objects (floating meshes) in Lethe have to be defined using triangular (simplex) meshes. Only triangular 2D meshes of 3D surfaces in 3D DEM simulations are presently supported. Quadrilateral 2D meshes of 3D surfaces and 1D mesh of 2D surfaces are not supported at the moment.
 
-This subsection explains the solid objects (floating meshes) information. First of all, the ``number of solids`` is specified. Then, for each solid object, we need a ``mesh`` subsection. In these subsections, the ``type``, ``initial refinement``, and other properties of the objects are defined. Note that ``simplex`` must be ``true`` for all the objects since the solid objects are defined using simplex meshes. For more information on mesh subsection, visit `CFD mesh <https://chaos-polymtl.github.io/lethe/documentation/parameters/cfd/mesh.html>`_
+This subsection explains the solid objects (floating meshes) information. First of all, the ``number of solids`` is specified. Then, for each solid object, we need a ``mesh`` subsection. In these subsections, the ``type``, ``initial refinement``, and other properties of the objects are defined. Note that currently, only ``two dimensions`` solid object are usable. The ``simplex`` parameter must be set to ``true`` for all the objects since 2D solid objects are defined using simplex meshes. For more information on mesh subsection, visit `CFD mesh <https://chaos-polymtl.github.io/lethe/documentation/parameters/cfd/mesh.html>`_
```

Comment:
I suggest we agree on the name of the floating mesh/solid object? what do you thing?
The page is "Floating Mesh (Solid Objects)" but here it says "...the solid objects (floating meshes)..." x)
We should choose one for good!

Reply:
I would use solid object exclusively. If we both agree on this, I could do a PR to fix this.

Reply:
It will needs many in-code changes tho. Floating wall also needs to change.
We can agree on Solid object and virtual wall :D ?

Reply:
Yup I agree with this. Virtual wall especially is a very good nomenclature.